### PR TITLE
Document pam settings

### PR DESCRIPTION
**Pull request by d-w-moore**
This pull request only documents pam related settings.  It really consists of only one commit, "document new legacy_auth.pam.* settings ", but is rebased to its proper position relative to PR 486 on which it depends.

To elaborate: 
Pull Requests
   - #486, 
   - #461 (having #486 as a prerequisite), and
   - this one (documenting #461)
   
should be merged in that exact order.

### CODE REVIEW

**Comment by d-w-moore** [comment]
@d-w-moore  Before taking out of draft - rebase to upstream/main

**Comment by trel** [comment]
looking pretty good.

**Comment by d-w-moore** [comment]
I'll undraft this when github stops giving me the angry unicorn head, and I can view the rendering of the `README.rst`

**Comment by alanking** [comment]
> I'll undraft this when github stops giving me the angry unicorn head, and I can view the rendering of the `README.rst`

I think that we've determined that this is not going to go away until something moves on GitHub's end, or we break up the README, or we move to Markdown, or... something else entirely. See #503.

Regardless, please undraft when ready for review. Thanks

**Comment by d-w-moore** [comment]
> > I'll undraft this when github stops giving me the angry unicorn head, and I can view the rendering of the `README.rst`
> 
> I think that we've determined that this is not going to go away until something moves on GitHub's end, or we break up the README, or we move to Markdown, or... something else entirely. See #503.
> 
> Regardless, please undraft when ready for review. Thanks

Looks good using the local renderer... https://github.com/irods/python-irodsclient/issues/503#issuecomment-1890911565

**Comment by trel** [comment]
looks good enough to squash to taste i think

File: README.rst
Comment (alanking) [reply README.rst:393]: Oh, good catch.

Comment (d-w-moore) [reply README.rst:393]: Ah,no should be write password to environment

Comment (d-w-moore) [reply README.rst:393]: Yeah that got all screwed up...hhm

**Review by alanking — COMMENTED** [review]
Looks better. I say squash, and leave off the #s for now.

**Review by korydraughn — COMMENTED** [review]
I agree with @alanking.

**Comment by korydraughn** [comment]
@d-w-moore Please mark the review comments resolved if completed.

File: README.rst
Comment (d-w-moore) [reply README.rst]: Ok, I believe this is corrected.

**Comment by d-w-moore** [comment]
> @d-w-moore Please mark the review comments resolved if completed.

done

**Review by korydraughn — APPROVED** [review]
Looks good.

Let's wait for one more approval before squashing/pounding.

**Comment by trel** [comment]
squash em.

**Comment by d-w-moore** [comment]
Squashed.

**Review by korydraughn — APPROVED** [review]
Commits look good. Please make sure issue numbers are correct and add the pounds.

**Comment by d-w-moore** [comment]
> Commits look good. Please make sure issue numbers are correct and add the pounds.

Done!

**Pull request merged**
